src/test/java/com/connexta/search/common/SearchTestData.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.search.common;
+
+import static com.connexta.search.common.configs.SolrConfiguration.*;
+import static com.connexta.search.common.configs.SolrConfiguration.CONTENTS_ATTRIBUTE;
+import static com.connexta.search.common.configs.SolrConfiguration.COUNTRY_CODE_ATTRIBUTE;
+import static com.connexta.search.common.configs.SolrConfiguration.CREATED_DATE_ATTRIBUTE;
+import static com.connexta.search.common.configs.SolrConfiguration.EXPIRATION_DATE_ATTRIBUTE;
+import static com.connexta.search.common.configs.SolrConfiguration.ICID_ATTRIBUTE;
+import static com.connexta.search.common.configs.SolrConfiguration.KEYWORD_ATTRIBUTE;
+import static com.connexta.search.common.configs.SolrConfiguration.MEDIA_TYPE_ATTRIBUTE;
+import static com.connexta.search.common.configs.SolrConfiguration.MODIFIED_ATTRIBUTE;
+import static com.connexta.search.common.configs.SolrConfiguration.RESOURCE_URI_ATTRIBUTE;
+import static com.connexta.search.common.configs.SolrConfiguration.TITLE_ATTRIBUTE;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
+
+public class SearchTestData {
+
+  @Nullable
+  public static String get(String key) {
+    return map.get(key);
+  }
+
+  public static final Map<String, String> map =
+      ImmutableMap.<String, String>builder()
```

Comment:
✏️ Java 9 added `Map.of()` for immutable maps.

Reply:
@peterhuffer and I looked at this and found a way to do it with Map.forEntries.